ibex_configs.hjson

```diff
@@ -0,0 +1,18 @@
+{
+  small-fastmult : {
```

Comment:
Please add a description field to tell users a bit more about this configuration in text form.